app/views/settings/index.html.erb

```diff
@@ -10,6 +11,11 @@
 <div class="tab-content">
   <% @settings.group_by(&:category).each do |category, setting| %>
     <div class="tab-pane <%= "active" if category == @settings.first.category %>" id='<%= short_cat(category) %>' >
+      <% if category == "Setting::Email" %>
```

Comment:
Adding a partial per setting type would be much better (see also, compute resources pages).